src/sentry/models/integration.py

```diff
@@ -13,6 +13,20 @@
 from sentry.signals import integration_added
 
 
+class PagerDutyServiceProject(Model):
+    __core__ = False
+
+    project = FlexibleForeignKey("sentry.Project", db_index=False, db_constraint=False)
```

Comment:
You'll want an index here. This could be handled by the unique constraint @scefali mentioned, if you make it `(project, organization_integration)`

Reply:
@wedamija I think that will depend on how @MeredithAnya implements it. If we query this table by getting all rows with a matching `organization_integration`, we don't need the `project` in the index. I suspect that's how it will be done but I'm not 100% Meredith is going to do it that way.

Reply:
We still need an index on project regardless. Important for if a project is deleted and we need to find the related rows to remove, etc.